app/transactions/members/update_transaction.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class Members::UpdateTransaction
+  include Dry::Transaction
+
+  step :extract_ids
```

Comment:
Since this step will never fail, you can use `tee` instead of `step`, to signify that this operation will happen everytime
```ruby
tee :extract_ids
```